test/videoelementfactory/NoOpVideoElementFactory.test.ts

```diff
@@ -43,5 +46,9 @@ describe('NoOpVideoElementFactory', () => {
       element.removeAttribute('name');
       expect(element.hasAttribute('name')).to.equal(false);
     });
+
+    it('can call play method', async () => {
+      await expect(element.play()).not.to.be.rejected;
```

Comment:
We are removing the `videoElement.play()` from under the `requiresVideoElementWorkaround()`, then do we need this unit test?

Reply:
This is to test the NoOpVideoElementFactory which is kinda a mock for video element. We might be able to just remove it or move it to test folder but well dont have time :D